### PR TITLE
Kyma Dashboard Version with link

### LINKED ISSUE #703: Busola Version uses hash and deep link
As [Sebastian](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#sebastian---tech-lead) and [Mathew](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#matthew---cloud-native-software-engineer) I would like to know what Busola Version I'm using. 

**Acceptance Criteria**

- [x] Follow Project DOD
- [x] Busola Version Contains the git hash from which the Busola was build. This is not the same as the Docker image hash
- [x] The git hash is deep linked with our Busola Repo
- [x] The name is changed to mention clearly that this is a Kyma Dashboard Version
- [x] Tests are present testing the functionality

**Tasks**

- [ ] Add Tasks

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:

- Changed `Version` to `Kyma Dashboard Version`
- Added link do version
- Added validation for commit, PR and dev version.

**Related issue(s)**
Closes #703

### CODE REVIEW

File: core/src/luigi-config/nav-footer.js [hunk 1]
```diff
@@ -34,3 +35,13 @@ async function getBusolaVersion() {
     .then(json => json.version)
     .catch(() => 'unknown');
 }
+
+function checkVersionLink(version) {
+  if (version !== 'dev') {
+    if (version.substring(0, 3) === 'PR-') {
```
Comment (Wawrzyn321) [inline core/src/luigi-config/nav-footer.js:41]: if (version.startsWith('PR-')) {

**Review by Wawrzyn321 — APPROVED** [review]

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**